mobile/src/main/res/layout/dialog_items_unavailable_warning.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="fill_parent"
+    android:layout_height="wrap_content" >
+
+    <CheckBox
+        android:id="@+id/checkbox_dont_show_again"
+        android:text="@string/dont_show_again"
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="5dp" />
```

Comment:
Stick to an 8dp baseline grid. Also make sure that it lines up with the message text in the dialog